gpx/src/commonMain/kotlin/org/maplibre/spatialk/gpx/serializers/InstantSerializer.kt

```diff
@@ -0,0 +1,35 @@
+package org.maplibre.spatialk.gpx.serializers
+
+import kotlin.time.ExperimentalTime
+import kotlin.time.Instant
+import kotlinx.datetime.LocalDateTime
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.toInstant
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+
+@OptIn(ExperimentalTime::class)
+internal object InstantSerializer : KSerializer<Instant> {
```

Comment:
I'd name it for the specific intended format, like Iso8601InstantSerializer in this case.

Though I'm curious, does the default serializer not do the same format?